CHANGELOG.md

```diff
@@ -14,6 +14,10 @@
     clients and then add the new scopes `ADMIN` and `ME`. See the
     [migration notes] (docs/Migration.md#from-22-to-30) for further details.
 
+- Allow colons (:) as field separators for URNs of extensions, since this is
+  what the SCIM specification defines. Using periods (.) is still possible,
+  but will log a warning message. 
+
```

Comment:
And isn't there an entry missing? Isn't it now possible to also send the full-qualified field name, if it is a core field, i.e. sending:

```
urn:ietf:params:scim:schemas:core:2.0:User:userName eq 'John'
```